.codesandbox/ci.json

```
@@ -10,5 +10,5 @@
     "packages/vitest",
     "packages/worker"
   ],
-  "node": "16"
+  "node": "20"
```

20 does not seem supported (more details at: https://codesandbox.io/docs/learn/ci#how-do-i-set-this-up)

```suggestion
  "node": "18"
```